src/ngraph/frontend/onnx_import/core/null_node.hpp

```diff
@@ -33,7 +33,7 @@ namespace ngraph
         ///
         /// More:
         /// https://github.com/onnx/onnx/blob/master/docs/IR.md#optional-inputs-and-outputs
-        class NullNode : public ngraph::Node
+        class NGRAPH_API NullNode : public ngraph::Node
```

Comment:
This change causes linker errors in ONNXRuntime. I've sent more details over email but I think this class can't use the `NGRAPH_API` decorator.

Reply:
Reverted the change.